docs/man_pages/publishing/publish-add.md

```diff
@@ -0,0 +1,21 @@
+publish add
```

Comment:
publish add
==========

Usage | Synopsis
------|-------
General | `$ appbuilder publish add [<Name> [<PublishUrl>]]`

Saves a new server connection. \<% if(isHtml) { %\>If you do not provide one or more command parameters, the Telerik AppBuilder CLI shows an interactive prompt to let you set the remaining details. 

\#\#\# Related Commands

Command | Description
----------|----------
[publish](publish.html) | Lists all saved connections or publishes to a connection/remote ftp server
[publish remove](publish-remove.html) | Removes a saved connection
\<% } %\>